### DISCUSSION TITLE
New user examples repository

### DISCUSSION THREAD

**Discussion by ruudvdent** [WAM2layers/WAM2layers#433] [Show and tell]
### My cool analysis with WAM2layers

Also feel free to share code and visualizations here in the [user-examples](https://github.com/WAM2layers/user-examples)

**Comment by P-Myers** [comment]
Dear WAM2layers Community & Developers,

Thank you all for being such an active and responsive group. Coming into this exciting research field with little previous experience, I struggled quite a bit, however I found the user manuals and tutorials provided very helpful. While by no means a perfect project, I have successfully completed my Master's thesis at the Eberswalde University for Sustainable Development (Germany). I would greatly appreciate your criticisms and insights on the work, as I know I have still much to learn...

The work has been published on the universities OPUS server, found here [urn:nbn:de:kobv:eb1-opus-10121](https://nbn-resolving.org/urn:nbn:de:kobv:eb1-opus-10121)

Cheers,

Parker Myers

**Reply by ruudvdent** [reply]
Dear Parker, 

That's awesome to hear! You have developed a couple of very nice plots in your thesis. If you're able to share the code of specifically Figures 4 and 5 it could be useful to other users. 

All the best!
